src/App.tsx

```diff
@@ -1,31 +1,28 @@
 import React, { useState } from "react";
 import "./App.css";
-import { ImportModal } from "./Components/ImportModal/ImportModal";
-import { Header } from "./Components/Shared/Header";
-import { Container } from "react-bootstrap";
-import { VideoModal } from "./Components/VideoModal/VideoModal";
 import type { MediaPresenter } from "./Components/MediaPresenter";
-
+import {Modal} from "./Components/Modal/Modal";
```

Comment:
styling: keep spacing consistent around the braces. (you have space before and after `MediaPresenter` on line 3 and not on line 4 & 5)